### DISCUSSION TITLE
ble names

### DISCUSSION THREAD

**Discussion by chaimt** [mKeRix/room-assistant#457] [Q&A]
Is there a way to give a friendly name to a mac address so I know what it is?

also is there a way to find what the mac address is (type of device)

thanks

**Comment by mKeRix** [comment]
If the devices includes its name in the advertisement room-assistant will automatically pick that up and use it for the entities. You can use [tag overrides](https://www.room-assistant.io/integrations/bluetooth-low-energy.html#settings) to set the names for devices that don't do that (or to rename those that do).

room-assistant does not offer a way to determine what a device type is from the advertisement it receives yet.
